### PR TITLE
Rewrite quote form to react-final-form

### PR DESCRIPTION
**Pull request by eikhr**
# Description

Rewrote from redux-form to react-final-form. Also changed from `TextAreas` to `TextInput`.

# Result

Before:
Screenshot 2023-02-18 at 21 07 30

After:
Screenshot 2023-02-18 at 21 07 49

# Testing

- [x] I have thoroughly tested my changes.

I added a quote

---

Resolves ABA-55

### CODE REVIEW

**Review by ollfkaih — APPROVED** [review]
LGTM

**Review by ivarnakken — APPROVED** [review]
:shipit: lgtm

File: app/routes/quotes/components/AddQuote.tsx [hunk 1]
```diff
@@ -1,95 +1,108 @@
...
 import { navigation } from '../utils';
 import styles from './Quotes.css';
 
 type Props = {
-  addQuotes: (arg0: Record<string, any>) => Promise<any>;
-  invalid: boolean;
-  pristine: boolean;
-  submitting: boolean;
-  handleSubmit: (arg0: (arg0: Record<string, any>) => Promise<any>) => void;
+  addQuotes: (quote: { text: string; source: string }) => Promise<unknown>;
```
Comment (LudvigHz) [inline app/routes/quotes/components/AddQuote.tsx:13]: Could you put the actual return type here? Or void if you're unsure. It's better then unknown when you know you don't use it IMO.

**Review by LudvigHz — APPROVED** [review]
LGTM

**Comment by LudvigHz** [comment]
I preferred the textarea though

**Comment by ivarnakken** [comment]
I need this 😬

**Pull request merged**